Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -100,7 +100,8 @@ configuration.
         --namespace kube-system \\
         --set global.kubeProxyReplacement=strict \\
         --set global.k8sServiceHost=API_SERVER_IP \\
-        --set global.k8sServicePort=API_SERVER_PORT
+        --set global.k8sServicePort=API_SERVER_PORT \\
+        --set global.ipam.operator.clusterPoolIPv4PodCIDR=10.217.0.0/16
```

Comment:
Thanks for the PR, this should be `--set config.ipam=kubernetes` instead of `--set global.ipam.operator.clusterPoolIPv4PodCIDR=10.217.0.0/16`

Reply:
@aanm Thank you for tell me that。 i'll deploy again